### PR TITLE
cnf-tests: improve sriov nic selector

### PR DESCRIPTION
**Pull request by SchSeba**
this commit add a check to be sure for intel nics that we enable sriov in the bios. if that is not the case the totalvf variabel will be 0 so we skip that nic

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

**Comment by SchSeba** [comment]
/hold

need to wait for https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/373

**Comment by SchSeba** [comment]
This one can continue new as we did the sriov bump

https://github.com/openshift-kni/cnf-features-deploy/pull/1323

File: cnf-tests/testsuites/e2esuite/dpdk/dpdk.go [hunk 1]
```diff
@@ -709,6 +709,14 @@ func findSriovDeviceForDPDK(sriovInfos *sriovcluster.EnabledNodes, nodeNames []s
 				if sriovInfos.IsSecureBootEnabled[nodeName] && iface.Vendor == networks.MlxVendorID {
 					continue
 				}
+
```
Comment (fedepaol) [inline cnf-tests/testsuites/e2esuite/dpdk/dpdk.go:712]: Can you wrap this in a "isIntelDisabledNic" function?

**Comment by SchSeba** [comment]
both issues are not related to this PR

```
[91m[1mSummarizing 2 Failures:[0m

[91m[1m[Fail] [0m[90m[tuningcni] [0m[0mtuningcni over macvlan [0m[91m[1m[It] pods with sysctl's over macvlan should be able to ping each other [0m
[37m/go/src/github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/security/tuning.go:93[0m

[91m[1m[Fail] [0m[90m[rfe_id:27368][performance] [0m[0mRPS configuration [0m[91m[1m[It] [test_id:55012] Should have the correct RPS configuration [0m
[37m/go/src/github.com/openshift-kni/cnf-features-deploy/vendor/github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/1_performance/performance.go:336[0m
```

/override ci/prow/e2e-gcp-ovn

**Pull request merged**